static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.tsx

```diff
@@ -294,6 +294,16 @@ function WidgetBuilder({
     if (notDashboardsOrigin) {
       fetchDashboards();
     }
+
+    if (widgetBuilderNewDesign) {
+      setState(prevState => ({
+        ...prevState,
+        selectedDashboard: {
+          label: dashboard.title,
+          value: dashboard.id === '' ? 'new' : dashboard.id,
```

Comment:
can we reuse https://github.com/getsentry/sentry/blob/cb5ef6104b569b2023826a44d78a4445e0be8390/static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.tsx#L85 here?